app/code/Magento/InventorySalesApi/etc/webapi.xml

```diff
@@ -19,4 +19,16 @@
             <resource ref="InventorySalesApi::stock"/>
         </resources>
     </route>
+    <route url="/V1/inventory/is-product-salable-for-requested-qty/:sku/:stockId/:requestedQty" method="GET">
+        <service class="Magento\InventorySalesApi\Api\IsProductSalableForRequestedQtyInterface" method="execute"/>
+        <resources>
+            <resource ref="InventorySalesApi::stock"/>
+        </resources>
+    </route>
+    <route url="/V1/inventory/is-product-salable/:sku/:stockId" method="GET">
+        <service class="Magento\InventorySalesApi\Api\IsProductSalableInterface" method="execute"/>
+        <resources>
+            <resource ref="InventorySalesApi::stock"/>
+        </resources>
+    </route>
```

Comment:
just noticed that you added web api service for already existing endpoint 
```
    <route url="/V1/inventory/is-product-salable/:sku/:stockId" method="GET">
        <service class="Magento\InventorySalesApi\Api\IsProductSalableInterface" method="execute"/>
        <resources>
            <resource ref="InventorySalesApi::stock"/>
        </resources>
    </route>
 ...
    <route url="/V1/inventory/is-product-salable/:sku/:stockId" method="GET">
        <service class="Magento\InventorySalesApi\Api\IsProductSalableInterface" method="execute"/>
        <resources>
            <resource ref="InventorySalesApi::stock"/>
        </resources>
    </route>
```

Reply:
I think you missed it with service `\Magento\InventorySalesApi\Api\StockResolverInterface::execute`

Reply:
Yes, definitely. Fixed in my last commit.

Reply:
Yes, definitely. Fixed in my last commit.

Reply:
Yes, definitely. Fixed in my last commit.